JSTask3.js

```diff
@@ -0,0 +1,21 @@
+// Write a JavaScript function to find the unique elements from two arrays Sample Array:
+// arr1 = [1, 2, 3, 4, 5];
+// arr2 = [1, 3, 8];
+// Expected Output : [2, 4, 5, 8]
+
+let arr1 = [1, 2, 3, 4, 5];
+let arr2 = [1, 3, 8];
+
+let arr3 = arr1.concat(arr2);
+console.log(arr3);
+
+// Remove Duplicate elements
+let unique = [...new Set(arr3)];
+console.log(unique);
+
+
+// Unique Elements
+let unique1 = arr1.filter(function(obj){
```

Comment:
Does not work. Please test the code. Expected output is [2, 4, 5, 8] but the output for the code is  [2, 4, 5]

![image](https://user-images.githubusercontent.com/22799565/211297165-ff31ca10-86bd-4067-97a7-d6c9a1c48dbd.png)